### PR TITLE
WPCOM Plugins: update notice type and tweak business plan

### PR DESCRIPTION
**Pull request by drw158**
Reasoning for adding "Coming Soon" items to business plan:

 Looks empty, doesn't look compelling. By adding filler items, it at least looks like we are working on making the plan better. Is it disingenuous though, because we don't have any concrete plan for adding value to business?

cc @rralian 

@adambbecker what do you think of the design changes?

## Before

9l3xqiz3gi7aaaaaelftksuqmcc

## After

ef0hdddixf8dmmmmgtkztpiaaaaasuvork5cyii

### CODE REVIEW

**Comment by rralian** [comment]
I think we do have differentiation in our business plan, it's just not necessarily in plugin form. I think the items shown here make sense (https://wordpress.com/plans/features/google-analytics/:site). I just don't know if they make sense on this page. 

I wouldn't do a coming soon section though. I don't think anyone would choose a plan based on something coming soon.

**Comment by drw158** [comment]
No, I don't think they make sense on this page, I'll revert.

**Comment by drw158** [comment]
The link is now external 

screen shot 2016-04-19 at 12 54 13 pm

File: client/my-sites/plugins-wpcom/info-header.jsx [hunk 1]
```diff
@@ -7,10 +7,11 @@ export const InfoHeader = React.createClass( {
 	render() {
 		return (
 			<Notice
+				status="is-info"
 				showDismiss={ false }
 				text={ "Your site comes pre-installed with a wide variety of plugins. Uploading your own plugins is not available on WordPress.com." }
 			>
-				<NoticeAction href="https://en.support.wordpress.com/plugins/">
+				<NoticeAction href="https://en.support.wordpress.com/plugins/" external="true">
```
Comment (rodrigoi) [inline client/my-sites/plugins-wpcom/info-header.jsx]: for consistency we should use `external={ true }`

**Comment by rodrigoi** [comment]
@drw158 you can :shipit: as soon as you fix that small inconsistency.

**Pull request merged**